core/kernel/classes/class.Resource.php

```diff
@@ -764,8 +758,7 @@ protected function toResource($value)
 
     private function onUpdate(): void
     {
-        /** @var EventAggregator $eventAggregator */
-        $eventAggregator = $this->getServiceManager()->get(EventAggregator::SERVICE_ID);
-        $eventAggregator->put($this->getUri(), new ResourceUpdated($this));
+        $eventManager = $this->getServiceManager()->get(EventManager::SERVICE_ID);
+        $eventManager->trigger(new ResourceUpdated($this));
```

Comment:
Now it executes the event, before It seems it just aggregated by URI, so when one URI is used, all the events are triggered. Will we have same behavior now? Why did we change it?

Reply:
We have changed it because it is deferring event and we do not know what it will imply, it is a breaking changes

Reply:
After @andreluizmachado answer would we keep the removal?

Reply:
I would say yes, he confirms that this service tries to improve the taskqueue mechanism and avoid multiple calls based on events. 
This workaround is good but the problem stay the same. I can close it, but number of triggered events will not changed (only the emition)
And, as discussed with @andreluizmachado , this fix is changing behavior of eventManager for the whole platform and can create side effect. Need to be confirmed by regression tests